src/util/fd_util_linux.c

```diff
@@ -0,0 +1,34 @@
+#define _GNU_SOURCE
+#include <errno.h>
+#include <fcntl.h>
+#include <sched.h>
+
+#include "fd_util_base.h"
+#include "log/fd_log.h"
+
+/* fd_linux_enter_netns: Replaces the network namespace of the calling
+   thread with the namespace located at the given nsfs mount path. */
+static void
+fd_linux_enter_netns( char const * netns ) {
```

Comment:
nit: I wonder if this function should propagate the error codes from the syscalls?